database/migrations/2024_07_05_140225_alter_site_polygon_table_to_add_versioning_attributes.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('site_polygon', function (Blueprint $table) {
+            $table->string('version_name')->nullable();
+            $table->string('primary_uuid')->nullable()->after('uuid');
+            $table->integer('is_active')->nullable();
```

Comment:
This field would be better represented in the code as a boolean. I'd update this to `$table->boolean('is_active')->default(false)` and change the code that's using `1` and `0` for `is_active` to use `true` and `false`. It'll also result in a smaller column size, because it'll define the column as a `tinyint(1)`